C/Cpp_include/c4Database.hh

```diff
@@ -37,6 +38,8 @@ C4_ASSUME_NONNULL_BEGIN
 C4EncryptionKey C4EncryptionKeyFromPassword(fleece::slice password,
                                             C4EncryptionAlgorithm = kC4EncryptionAES256);
 
+/// Derives an encryption key from a user-entered password with SHA1 as the hashing function
+std::array<uint8_t,32> C4EncryptionKeyFromPasswordSHA1(fleece::slice password);
```

Comment:
Should it be the same signature as C4EncryptionKeyFromPassword but include SHA1 in C4EncryptionAlgorithm?